packages/block-editor/src/components/button-block-appender/index.native.js

```diff
@@ -26,12 +30,12 @@ function ButtonBlockAppender( { rootClientId } ) {
 						disabled={ disabled }
 						fixedRatio={ false }
 					>
-						<View style={ [ styles.appender, { flex: 1 } ] }>
+						<View style={ [ appenderStyle, { flex: 1 } ] }>
```

Comment:
WDYT about moving this `flex:1` to styles?

Reply:
make sense